features/bootstrap/SpeakerContext.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+use Behat\MinkExtension\Context\RawMinkContext;
+use App\DataFixtures\SpeakerFixtures;
+
+final class SpeakerContext extends RawMinkContext
+{
+    private const PATH_SPEAKER = '/speakers/';
+    
+    /**
+     * @When /^I am on the speaker listing page$/
+     */
+    public function iAmOnTheSpaekerListingPage(): void
+    {
+        $this->visitPath(self::PATH_SPEAKER);
+    }
+
+    /**
+     * @When /^I am on the speaker creation page$/
+     */
+    public function iAmOnTheSpeakerCreationPage(): void
+    {
+        $this->visitPath(self::PATH_SPEAKER.'create');
+    }
+
+    /**
+     * @Then /^I should be redirected on the speaker listing page$/
+     */
+    public function iShouldBeRedirectedOnTheSpeakerListingPage(): void
+    {
+        $this->assertSession()->addressEquals($this->locatePath(self::PATH_SPEAKER));
+    }
+
+    /**
+     * @When /^I fill the speaker "([^"]*)" field with "([^"]*)"$/
+     */
+    public function iFillTheSpeakerFieldWith(string $field, string $value): void
+    {
+        if ("speaker" === $field) {
+            if (isset(SpeakerFixtures::DEFAULT_SPEAKER[$value])) {
```

Comment:
Use the opposite condition and throw the exception, then put the content of this one as main execution.
This way you will remove the `else`